pkg/sql/logictest/logic.go

```diff
@@ -1708,6 +1739,73 @@ func (t *logicTest) openDB(pgURL url.URL) *gosql.DB {
 	return gosql.OpenDB(connector)
 }
 
+func (t *logicTest) newTestServerCluster(binaryPath string) {
+	var localBinary string
+	var upgradeBinary string
+	var found bool
+	if bazel.BuiltWithBazel() {
+		localBinary, found = bazel.FindBinary("pkg/cmd/cockroach", "cockroach")
```

Comment:
We should probably also check for `cockroach-short`.

Reply:
On a second thought, `findBinary` [1] already does exactly what you need, no? We should probably move it outside of roachtest, possibly into `pkg/testutils/` and reuse it.

[1] https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/cluster.go#L117